tests/python/unittest/test_symbol.py

```diff
@@ -115,7 +115,7 @@ def test_symbol_infer_shape_var():
     shape = (2, 3)
     a = mx.symbol.Variable('a', shape=shape)
     b = mx.symbol.Variable('b')
-    c = a+b
+    c = mx.symbol.ewise_plus(a, b)
```

Comment:
strictly it relies on backward inference to get shape of b, as per other argument,  since operator+ already defaults to broadcast add, consistency is needed